appcompat-theme/build.gradle

```diff
@@ -25,12 +25,12 @@ kotlin {
 }
 
 android {
-    compileSdkVersion 32
+    compileSdkVersion 33
```

Comment:
The latest API version we test with in CI is 29, we should probably update that at some point to get closer to latest.

Reply:
Great point. I will do that post upgrade